### PR TITLE
Changed documentation design

### PR DESCRIPTION
**Pull request by TheOneMusic**
## Summary

I noticed two little problems with the documentation design.

The first one was with the [`Permissions.all_channel()`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Permissions.all_channel) class method, where all the permissions were stacked up together instead of being separated by a newline.

The second one was with the [`Permissions.advanced()`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Permissions.advanced) class method, where **New in version 1.7** didn't appear.

This pull request is to fix those two issues.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/permissions.py [hunk 1]
```diff
@@ -149,15 +149,16 @@ def all_channel(cls):
...
-        - view_audit_log
-        - view_guild_insights
-        - manage_guild
-        - change_nickname
-        - manage_nicknames
-        - kick_members
-        - ban_members
-        - administrator
+        
+        - :attr:`manage_emojis`\n
```
Comment (ShineyDev) [inline discord/permissions.py:153]:
```suggestion
        * :attr:`manage_emojis`
```

**Review by ShineyDev — CHANGES_REQUESTED** [review]

File: discord/permissions.py (hunk 1 above)
Comment (TheOneMusic) [reply discord/permissions.py:153]: @ShineyDev Thank you for this, but why use an asterix?

Comment (ShineyDev) [reply discord/permissions.py:153]: You're right. This was caused in #6476. Just remove the newlines.

**Review by Rapptz — CHANGES_REQUESTED** [review]
There's no reason for those \n characters to be there.

**Comment by TheOneMusic** [comment]
@Rapptz @ShineyDev Thank you for your advice. But why wasn't the newline characters appearing in the `discord.py` documentation page?

**Comment by ShineyDev** [comment]
There was a missing newline before the bulleted list block started.

**Comment by TheOneMusic** [comment]
Okay I see thank you. I removed the newlines

**Review by Rapptz — APPROVED** [review]
LGTM.

**Pull request merged**